### PR TITLE
feat(docker): add docker files and config

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes https://github.com/mozilla/fxa-oauth-console/issues/43

### CODE REVIEW

File: Dockerfile-build [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+COPY package.json package.json
+
+RUN npm install --production && rm -rf ~app/.npm /tmp/*
+
+COPY . /app
+USER root
+RUN chown app:app /app/config
+
+USER root
+RUN chown root:root /app/config
```
Comment (jrgm) [inline Dockerfile-build]: Duplicate lines?

File: Dockerfile-build [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FROM node:4.8.2-alpine
+
+RUN apk add --no-cache git make gcc g++ linux-headers openssl python
```
Comment (jbuck) [inline Dockerfile-build]: Does this app have native dependencies that require these build utilities?

File: Dockerfile-build [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+WORKDIR /app
+
+USER app
+
+COPY package.json package.json
+
+RUN npm install --production && rm -rf ~app/.npm /tmp/*
+
+COPY . /app
+USER root
```
Comment (jbuck) [inline Dockerfile-build]: The `USER` is already root at this point, so you could remove this line.

**Comment by jbuck** [comment]
@vladikoff I've added docker hub credentials to the circle build

**Comment by vladikoff** [comment]
> @vladikoff I've added docker hub credentials to the circle build

Thanks!!!

**Pull request closed without merging**